Add a basic getting started manifest example (#198)
DataDog/datadog-cloudfoundry-buildpack: README.md

* Datadog Trace Agent for submitting APM traces from your application
* Datadog IoT Agent for submitting your application logs

## Basic Usage

> If you don't have this buildpack installed in your foundry, you can reference one

Reviewer: ```suggestion
If you don't have this buildpack uploaded in your foundation, you can reference one
```

## Installation

Since this is a supply buildpack, it has to be specified before any final buildpack in the list. See [Cloud Foundry documentation][4] for details about pushing an application with multiple buildpacks.

> ### Option 1: Using a GitHub Release Packaged Zip File

Reviewer 2: ```suggestion
### Option 1: Using a GitHub release packaged as a ZIP File
```

> You can reference one of the GitHub release zip files directly in your `manifest.yml`:

Reviewer 2: ```suggestion
You can reference one of the GitHub release ZIP files directly in your `manifest.yml`:
```

```
---
applications:
- name: test-python-flask
  random-route: true
  buildpacks:
    - https://github.com/DataDog/datadog-cloudfoundry-buildpack/releases/download/4.42.0/datadog-cloudfoundry-buildpack-4.42.0.zip
    - python_buildpack
  memory: 256M
  stack: cflinuxfs4
  env:
    DD_API_KEY: <DATADOG_API_KEY>  
```

> ### Option 2: Upload the buildpack to CF

Reviewer 2: ```suggestion
### Option 2: Upload the buildpack to Cloud Foundry
```

    ```bash
    cf update-buildpack datadog-cloudfoundry-buildpack -p datadog-cloudfoundry-buildpack.zip
    ```

> Once it is available in your Cloud Foundry environment, configure your application to use the Datadog buildpack by specifying it in your application manifest.

Reviewer 2: ```suggestion
After it is available in your Cloud Foundry environment, configure your application to use the Datadog buildpack by specifying it in your application manifest.
```
